src/Google.Maps/Places/PlacesService.cs

```diff
@@ -67,7 +66,9 @@ public PlacesService(Uri baseUri)
 		public AutocompleteResponse GetAutocompleteResponse(AutocompleteRequest request)
 		{
 			var url = new Uri(this.BaseUri, request.ToUri());
-			return Internal.Http.Get(url).As<AutocompleteResponse>();
+
+			var http = new Internal.MapsHttp();

```

Comment:
wouldn't it make sense to have this declared at class level and re-use? That would have the knock on effect that we are re-using the same httpclient